tests/system-tests/rdscore/internal/rdscorecommon/sriov-pod-level-bond.go

```diff
@@ -0,0 +1,1114 @@
+package rdscorecommon
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"gopkg.in/k8snetworkplumbingwg/multus-cni.v4/pkg/types"
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/apiobjectshelper"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	. "github.com/openshift-kni/eco-gotests/tests/system-tests/rdscore/internal/rdscoreinittools"
+)
+
+const (
+	podLevelBondDeploymentRBACName = "privileged-rdscore-pod-level-bond"
+	podLevelBondDeploymentRBACRole = "system:openshift:scc:privileged"
+	podLevelBondDeploymentSAName   = "rdscore-pod-level-bond-sa"
+	podLevelBondPodLabel           = "systemtest-test=rdscore-pod-level-bond-privileged"
+	podLevelBondNetName            = "bond-net"
+	tcpTestPassedMsg               = `TCP test passed as expected`
+)
+
+var (
+	podLevelBondPodLabelMap = map[string]string{"systemtest-test": "rdscore-pod-level-bond-privileged"}
+)
+
+func createPrivilegedPodLevelBondDeployment(
+	apiClient *clients.Settings,
+	deploymentName,
+	nsName,
+	podLabel,
+	scheduleOnHost,
+	sriovNet1Name,
+	sriovNet2Name,
+	bondNetName,
+	bondInfIPv4,
+	bondInfIPv6,
+	bondInfSubMaskIPv4,
+	bondInfSubMaskIPv6,
+	bondInfMacAddr string) error {
+	glog.V(100).Infof("Creating the pod-level bonded deployment test source deployment")
```

Comment:
The next line seems to cleanup the deployment:
```
err := cleanUpPodLevelBondDeployment(apiClient, deploymentName, nsName, podLabel)
```

so message is a bit misleading
```suggestion
	glog.V(100).Infof("Ensuring deployment %q doesn't exist in %q namespace", deploymentName, nsName)
```